pkgs/applications/misc/kitty/default.nix

```diff
@@ -54,8 +100,13 @@ buildPythonApplication rec {
   '';
 
   postInstall = ''
+    terminfo_src=${if stdenv.isDarwin then
+      ''"$out/Applications/kitty.app/Contents/Resources/terminfo"''
+      else
+      "$out/share/terminfo"}
+
     mkdir -p $terminfo/share
-    mv $out/share/terminfo $terminfo/share/terminfo
+    mv "$terminfo_src" $terminfo/share/terminfo
```

Comment:
Does the app pick up the terminfo dirs?

Reply:
> Does the app pick up the terminfo dirs?

Are you asking if terminfo files are correctly installed? I'm not sure if the terminfo stuff is properly installed, but the files do exist:

```
$ cat /nix/store/nqyzdmi844za1vqqjh7n2s719ma15y9f-kitty-0.13.3/nix-support/propagated-user-env-packages
/nix/store/frhw04kcjzf0wfz3kby0ahkja10wh8rw-kitty-0.13.3-terminfo
$ find /nix/store/frhw04kcjzf0wfz3kby0ahkja10wh8rw-kitty-0.13.3-terminfo
/nix/store/frhw04kcjzf0wfz3kby0ahkja10wh8rw-kitty-0.13.3-terminfo
/nix/store/frhw04kcjzf0wfz3kby0ahkja10wh8rw-kitty-0.13.3-terminfo/share
/nix/store/frhw04kcjzf0wfz3kby0ahkja10wh8rw-kitty-0.13.3-terminfo/share/terminfo
/nix/store/frhw04kcjzf0wfz3kby0ahkja10wh8rw-kitty-0.13.3-terminfo/share/terminfo/78
/nix/store/frhw04kcjzf0wfz3kby0ahkja10wh8rw-kitty-0.13.3-terminfo/share/terminfo/78/xterm-kitty
```

Do you know how to verify that these terminfo files are working?

Note: kitty's build system is a bit weird. On macOS, it renames the UNIX `share` directory to `kitty.app/Contents/Resources`.